### ISSUE TITLE
https://elements.polymer-project.org/elements/iron-ajax#response

### ISSUE THREAD

**Issue by ebidel** [Polymer/polymer-element-catalog#233]
_From @gertcuykens on January 5, 2016 15:3_

https://elements.polymer-project.org/elements/iron-ajax#response

Documents for events should mention the stuff they shuffle into `detail` for example `detail.xhr.response`

![image](https://cloud.githubusercontent.com/assets/863731/12118155/307c60ce-b3c5-11e5-8d4d-c1c5c0b642c3.png)

PS and remove the returns part 

``` Returns {url: string, method: (string|undefined), async: (boolean|undefined), body: (ArrayBuffer|ArrayBufferView|Blob|Document|FormData|null|string|undefined|Object), headers: (Object|undefined), handleAs: (string|undefined), jsonPrefix: (string|undefined), withCredentials: (boolean|undefined)} ```
 
The signature is good enough, no need to clutter the docs, just do this

```toRequestOptions() ➙ {url: string, method: (string|undefined), async: (boolean|undefined), body: (ArrayBuffer|ArrayBufferView|Blob|Document|FormData|null|string|undefined|Object), headers: (Object|undefined), handleAs: (string|undefined), jsonPrefix: (string|undefined), withCredentials: (boolean|undefined)}```

_Copied from original issue: Polymer/docs#1484_

**Comment by blasten** [comment]
@ebidel I think the issue is in iron-ajax. https://github.com/PolymerElements/iron-ajax/blob/master/iron-ajax.html#L45-L62

**Comment by blasten** [comment]
Also, I'm not sure if there's a way to describe the event `detail` property other than this way: https://github.com/PolymerLabs/app-layout/blob/master/app-header/app-header.html#L409-L414 cc @garlicnation as he's the expert.

**Comment by ebidel** [comment]
Yea agreed. Some elements are lacking the details description, some have Closure style: https://github.com/PolymerElements/seed-element/blob/master/seed-element.html#L137

General issue may be a hydrolysis one: https://github.com/Polymer/hydrolysis/issues/103

Regardless, we're not consistent with how events are documented. I've updated this description to better match the report.

**Comment by blasten** [comment]
+1 I agree. Consistency, consistency!

**Comment by blasten** [comment]
I just tested the iron-component-page for seed-element and I saw that hydrolysis did parse the annotation.
```js
   {"tag":"detail","type":null,"description":"{{sound: String}}"}]]
```
@garlicnation is this the way going forward?  If so, we should update iron-component-page, so it shows the detail in the docs.
